samples/bluetooth/fast_pair/input_device/README.rst

```diff
@@ -78,6 +78,9 @@ See :ref:`ug_bt_fast_pair_provisioning` in the Fast Pair user guide for details.
    :start-after: nrf_desktop_fastpair_important_start
    :end-before: nrf_desktop_fastpair_important_end
 
+.. important::
```

Comment:
We could move this "admonitions" after line no: 74 because now we have 3 admonitions one after the other in the documentation.

Reply:
I decided to move it before the part where we describe the default model as the note is only relevant when the user decides to use their device model instead of the default Nordic model